microcosm_flask/conventions/alias.py

```diff
@@ -0,0 +1,47 @@
+"""
+Conventions for aliasing endpoints.
+
+"""
+from flask import redirect
+from microcosm_flask.conventions.base import Convention
+from microcosm_flask.naming import name_for
+from microcosm_flask.operations import Operation
+
+
+class AliasConvention(Convention):
+
+    def configure_alias(self, ns, definition):
+        """
+        Register an alias endpoint which will redirect to a resource's retrieve endpoint.
+
+        Note that the retrieve endpoint MUST be registered prior to the alias endpoint.
```

Comment:
There are two reasons for this condition:
 1. The value of `ns.alias_path` (used by Alias) will shadow the value of `ns.instance_path` (used by Retrieve); if we get the order wrong, Flask will probably invoke the wrong function.
 2. The implementation on L34 looks up the Retrieve endpoint and resolves its URL. So it must exist.